tinygrad/runtime/ops_python.py

```diff
@@ -137,6 +147,8 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
                 else: ret.append(_load(m, ox*4 + oy*dtp[0].shape[1]*4 + j))
               ul[i].append(ret)
           elif dtype.sz > 1:
+            if len(inp) == 4 and dtp[3].sz > 1:
```

Comment:
Hmm, this looks sketchy.

Reply:
Yea I don't think this is right

Reply:
Yes, I'm looking into it right now.

Reply:
I changed this in my second to last commit. The problem I was trying to solve is when gated `UOps.LOAD` reference values that have `.sz` > 1. I addressed that by sending the `gate`/`default` that corresponds to the current `UOps.LOAD` item.